src/QSymbolicsBase/predefined.jl

```diff
@@ -143,6 +174,12 @@ const Pm = const σ₋ = PauliM()
 const Pp = const σ₊ = PauliP()
 """Hadamard gate"""
 const H = HGate()
+"""Rotation around the X axis"""
+const Rx = RotXGate
+"""Rotation around the Y axis"""
+const Ry = RotYGate
+"""Rotation around the Z axis"""
+const Rz = RotZGate
```

Comment:
These aliases seem a bit out of place here. All the other consts here are not aliases, rather instances of types. Aliases could be useful, but for now let's just stick to only the types, to avoid accidental complexity.